gtsam/geometry/Cal3Bundler.cpp

```diff
@@ -118,6 +121,25 @@ Point2 Cal3Bundler::calibrate(const Point2& pi, const double tol) const {
     throw std::runtime_error(
         "Cal3Bundler::calibrate fails to converge. need a better initialization");
 
+  x = invKPi.x(), y = invKPi.y();
+  const double xx = x * x, yy = y * y;
```

Comment:
My suggestion was to take rr and g out of the loop above and make them non-const